README.md

```diff
@@ -34,15 +56,10 @@ if (! Electron.isDesktop){
 }
 ```
 
-## Download Button
-
-Add a download button to your app with `{{> electronDownload}}`. This button points to `/download/darwin-x64`, `/download/linux-ia64`, etc. based on platform. Feel free to point to these locations manually yourself as well.
-
-The download button will not be shown within the Electron application.
-
 ## Deploy
 
```

Comment:
Might still be nice to have helpers and a standard place to put the apps even if we're not building on production.